docs/docs/basic-usage.md

```diff
@@ -64,6 +64,24 @@ $ poetry add pendulum
 It will automatically find a suitable version constraint **and install** the package and subdependencies.
 
 
+## Using your virtual environment
+
+By default, poetry creates a virtual environment in `{cache-dir}/virtualenvs` (`{cache-dir}\virtualenvs` on Windows).
+There are several ways to run commands within this virtual environment.
+
+### Using `poetry run`
+
+To run your script simply use `poetry run python your_script.py`.
+Likewise if you have command line tools such as `pytest` or `black` you can run them using `poetry run pytest`.
+
+### Activating the virtual environment
+
+The easiest way to activate the virtual environment is `poetry shell`.
```

Comment:
Could you please add a hint, that `poetry shell` will spawn a new shell and activates there the venv? This is confusing a lot of people. Because you have to type `exit` to leave this shell and deactivate the venv.

Reply:
You can use the details here regarding how we need to create a sub-shell in order to activate the environment from within a python application.

https://github.com/python-poetry/poetry/issues/2792#issuecomment-672730038